src/components/Battles.ts

```diff
@@ -326,6 +326,30 @@ export class Battles<TGameStartr extends FullScreenPokemon> extends Component<TG
     }
 
     /**
+     * Tests to see if all party Pokemon have fainted.
+     * @returns If a player's party is wiped or not.
+     */
+    public isPartyWiped(): boolean {
+        for (const chosenPokemon of this.gameStarter.itemsHolder.getItem("PokemonInParty")) {
+            if (chosenPokemon.statistics.health.current !== 0) {
+
```

Comment:
I would consider this one of the places where it doesn't make sense to have blank line (confusing, I know).